tools/dev/run_dev_server.sh

```diff
@@ -22,6 +22,7 @@ export DISABLE_TELEMETRY=true # disable telemetry for local development
 function go_run() {
   GIT_HASH=$(git rev-parse --short HEAD)
   go run -ldflags "-X github.com/weaviate/weaviate/usecases/config.GitHash=$GIT_HASH" "$@"
+  go run -ldflags "-X github.com/weaviate/weaviate/usecases/config.IMAGE_TAG=localhost"
```

Comment:
You copied the line twice, instead you probable want to move the `-X ...` arg to line 24 :)

Reply:
good catch